meson.build

```diff
@@ -48,10 +48,45 @@ targets = []
 
 cc = meson.get_compiler('c')
 fs = import('fs')
+
+# Find the picolibc name for the host cpu. Provide
+# for some common aliases
+
+cpu_family_aliases = {
+		       # aarch64
+		       'arm64' : 'aarch64',
+		       # cris
+		       'crisv32' : 'cris',
+		       # i386
+		       'i486' : 'i386',
+		       'i586' : 'i386',
+		       'i686' : 'i386',
+		       'x86' : 'i386',
+		       # m68hc11
+		       'm6811' : 'm68hc11',
+		       'm6812' : 'm68hc11',
+		       'm68hc12' : 'm68hc11',
+		       # m68k
+		       'fido' : 'm68k',
+		       # m88k
+		       'm88110' : 'm88k',
+		       # or1k
+		       'or1knd': 'or1k',
```

Comment:
We don't even support them.  shouldn't we fail?

Reply:
It's a good thought -- maybe prune this list down to the supported archs and raise an error if it isn't found? We could leave the archs which have code ready to port in a  separate list and let people know when they've selected an arch which just hasn't had picolibc build support added?